packages/headless/src/features/commerce/common/actions.ts

```diff
@@ -60,7 +66,13 @@ export const buildBaseCommerceAPIRequest = async (
     context: {
       user,
       view,
-      cart: state.cart.cartItems.map((id) => state.cart.cart[id]),
+      cart: state.cart.cartItems.map((id) => {
```

Comment:
We were sending the product name and price, which are not in the API request body model, so I removed those.